tests/functional-tests/README.md

```diff
@@ -1,11 +1,8 @@
-# IBM Spectrum Scale CSI - Automation
+# Functional Test Automation Suite
 
-The IBM Spectrum Scale Container Storage Interface (CSI) project enables container orchestrators, such as Kubernetes and OpenShift, to manage the life-cycle of persistent storage.
+This Functional Test Automation Suite should be exercise and test the IBM Spectrum Scale CSI driver & operator functionality 
```

Comment:
Nit picky comment... some slight changes... suggest we can drop "should be" from this sentence and change "exercise" to "exercises and tests" ... so that if content changes in this repo for whatever reason, the statement is general enough to not require modification. 


> This Functional Test Automation Suite exercises and tests the IBM Spectrum Scale CSI functionality.